src/index.js

```diff
@@ -3,4 +3,8 @@ export { default as ElementTemplatesPropertiesProviderModule } from './element-t
 
 // utils
 export { Validator as CloudElementTemplatesValidator } from './cloud-element-templates/Validator';
-export { ElementTemplateLinterPlugin as CloudElementTemplatesLinterPlugin } from './cloud-element-templates/LinterPlugin';
+export { ElementTemplateLinterPlugin as CloudElementTemplatesLinterPlugin } from './cloud-element-templates/linting';
+
+// core
+export { default as CloudElementTemplatesCoreModule } from './cloud-element-templates/core';
```

Comment:
Looks good to me as a great start :drum:.

My assumption in the past was that we need a `core` only export for this to work, too. Let's see if that is what we end up with.

Reply:
As long as we don't bundle frontend dependencies, I think we should be safe.

Reply:
You were right, Nico: https://github.com/bpmn-io/element-templates-cli/pull/14

Reply:
Proposal: Provide a separate `core` export that allows me to consume the core (as is).